File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ems/uiemsdto/d/EmsPathgyItemViewDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.ems.uiemsdto.d
{
    /// <summary>
    /// EmsPathgyItemViewDTO 的摘要说明。
    /// </summary>
    public class EmsPathgyItemViewDTO : BaseDTO {

        public EmsPathgyItemViewDTO() {
 
        }

        /// <summary>
        /// 标本id
        /// </summary>
		public string Id_lagsamp {
            get { return getAttrVal<string>("Id_lagsamp",null); }
            set { setAttrVal<string>("Id_lagsamp", value); }
        }

        /// <summary>
        /// 标本名称
        /// </summary>
		public string Name_labsamp {
            get { return getAttrVal<string>("Name_labsamp",null); }
            set { setAttrVal<string>("Name_labsamp", value); }
        }

        /// <summary>
        /// 采集部位ID
        /// </summary>
		public string Id_body_coll {
            get { return getAttrVal<string>("Id_body_coll",null); }
            set { setAttrVal<string>("Id_body_coll", value); }
        }

        /// <summary>
        /// 采集部位SD
        /// </summary>
		public string Sd_body_coll {
            get { return getAttrVal<string>("Sd_body_coll",null); }
            set { setAttrVal<string>("Sd_body_coll", value); }
        }

        /// <summary>
        /// 采集部位
        /// </summary>
		public string Name_body_coll {
            get { return getAttrVal<string>("Name_body_coll",null); }
            set { setAttrVal<string>("Name_body_coll", value); }
        }

        /// <summary>
        /// 标本数量
        /// </summary>
		public string Quan_coll {
            get { return getAttrVal<string>("Quan_coll",null); }
            set { setAttrVal<string>("Quan_coll", value); }
        }

        /// <summary>
        /// 固定液ID
        /// </summary>
		public string Id_fixative {
            get { return getAttrVal<string>("Id_fixative",null); }
            set { setAttrVal<string>("Id_fixative", value); }
        }

        /// <summary>
        /// 固定液SD
        /// </summary>
		public string Sd_fixative {
            get { return getAttrVal<string>("Sd_fixative",null); }
            set { setAttrVal<string>("Sd_fixative", value); }
        }

        /// <summary>
        /// 固定液
        /// </summary>
		public string Name_fixative {
            get { return getAttrVal<string>("Name_fixative",null); }
            set { setAttrVal<string>("Name_fixative", value); }
        }

        /// <summary>
        /// 序号
        /// </summary>
		public string Sortno {
            get { return getAttrVal<string>("Sortno",null); }
            set { setAttrVal<string>("Sortno", value); }
        }

        /// <summary>
        /// 病理标本ID
        /// </summary>
		public string Id_appathgysamp {
            get { return getAttrVal<string>("Id_appathgysamp",null); }
            set { setAttrVal<string>("Id_appathgysamp", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_lagsamp", "Name_labsamp", "Id_body_coll", "Sd_body_coll", "Name_body_coll", "Quan_coll", "Id_fixative", "Sd_fixative", "Name_fixative", "Sortno", "Id_appathgysamp"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.ems.uiemsdto.d.EmsPathgyItemViewDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ems/uiemsdto/d/EmsPathgyItemViewDTO.java
package iih.ci.ord.dto.ems.uiemsdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 病理医疗单子 DTO数据 
 * 
 */
public class EmsPathgyItemViewDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 标本id
	 * @return String
	 */
	public String getId_lagsamp() {
		return ((String) getAttrVal("Id_lagsamp"));
	}
	/**
	 * 标本id
	 * @param Id_lagsamp
	 */
	public void setId_lagsamp(String Id_lagsamp) {
		setAttrVal("Id_lagsamp", Id_lagsamp);
	}
	/**
	 * 标本名称
	 * @return String
	 */
	public String getName_labsamp() {
		return ((String) getAttrVal("Name_labsamp"));
	}
	/**
	 * 标本名称
	 * @param Name_labsamp
	 */
	public void setName_labsamp(String Name_labsamp) {
		setAttrVal("Name_labsamp", Name_labsamp);
	}
	/**
	 * 采集部位ID
	 * @return String
	 */
	public String getId_body_coll() {
		return ((String) getAttrVal("Id_body_coll"));
	}
	/**
	 * 采集部位ID
	 * @param Id_body_coll
	 */
	public void setId_body_coll(String Id_body_coll) {
		setAttrVal("Id_body_coll", Id_body_coll);
	}
	/**
	 * 采集部位SD
	 * @return String
	 */
	public String getSd_body_coll() {
		return ((String) getAttrVal("Sd_body_coll"));
	}
	/**
	 * 采集部位SD
	 * @param Sd_body_coll
	 */
	public void setSd_body_coll(String Sd_body_coll) {
		setAttrVal("Sd_body_coll", Sd_body_coll);
	}
	/**
	 * 采集部位
	 * @return String
	 */
	public String getName_body_coll() {
		return ((String) getAttrVal("Name_body_coll"));
	}
	/**
	 * 采集部位
	 * @param Name_body_coll
	 */
	public void setName_body_coll(String Name_body_coll) {
		setAttrVal("Name_body_coll", Name_body_coll);
	}
	/**
	 * 标本数量
	 * @return String
	 */
	public String getQuan_coll() {
		return ((String) getAttrVal("Quan_coll"));
	}
	/**
	 * 标本数量
	 * @param Quan_coll
	 */
	public void setQuan_coll(String Quan_coll) {
		setAttrVal("Quan_coll", Quan_coll);
	}
	/**
	 * 固定液ID
	 * @return String
	 */
	public String getId_fixative() {
		return ((String) getAttrVal("Id_fixative"));
	}
	/**
	 * 固定液ID
	 * @param Id_fixative
	 */
	public void setId_fixative(String Id_fixative) {
		setAttrVal("Id_fixative", Id_fixative);
	}
	/**
	 * 固定液SD
	 * @return String
	 */
	public String getSd_fixative() {
		return ((String) getAttrVal("Sd_fixative"));
	}
	/**
	 * 固定液SD
	 * @param Sd_fixative
	 */
	public void setSd_fixative(String Sd_fixative) {
		setAttrVal("Sd_fixative", Sd_fixative);
	}
	/**
	 * 固定液
	 * @return String
	 */
	public String getName_fixative() {
		return ((String) getAttrVal("Name_fixative"));
	}
	/**
	 * 固定液
	 * @param Name_fixative
	 */
	public void setName_fixative(String Name_fixative) {
		setAttrVal("Name_fixative", Name_fixative);
	}
	/**
	 * 序号
	 * @return String
	 */
	public String getSortno() {
		return ((String) getAttrVal("Sortno"));
	}
	/**
	 * 序号
	 * @param Sortno
	 */
	public void setSortno(String Sortno) {
		setAttrVal("Sortno", Sortno);
	}
	/**
	 * 病理标本ID
	 * @return String
	 */
	public String getId_appathgysamp() {
		return ((String) getAttrVal("Id_appathgysamp"));
	}
	/**
	 * 病理标本ID
	 * @param Id_appathgysamp
	 */
	public void setId_appathgysamp(String Id_appathgysamp) {
		setAttrVal("Id_appathgysamp", Id_appathgysamp);
	}
}